File: src/main/kotlin/com/prince/skyblocksandbox/skyblockutils/ActionBarColor.java
package com.prince.skyblocksandbox.skyblockutils;

import org.bukkit.ChatColor;

import java.util.Locale;

@SuppressWarnings("unused")
enum ActionBarColor{
    BLACK,
    DARK_BLUE,
    DARK_GREEN,
    DARK_AQUA,
    DARK_RED,
    DARK_PURPLE,
    GOLD,
    GRAY,
    DARK_GRAY,
    BLUE,
    GREEN,
    AQUA,
    RED,
    LIGHT_PURPLE,
    YELLOW,
    WHITE,
    MAGIC,
    BOLD,
    STRIKETHROUGH,
    UNDERLINE,
    ITALIC,
    RESET
    ;

    public ChatColor asBukkit(){
        return ChatColor.valueOf(name());
    }

    @Override
    public String toString() {
        return name().toLowerCase(Locale.ENGLISH);
    }
}

File: src/main/kotlin/com/prince/skyblocksandbox/skyblockutils/ActionBarColor.kt
package com.prince.skyblocksandbox.skyblockutils

import org.bukkit.ChatColor

enum class ActionBarColor{
    BLACK,
    DARK_BLUE,
    DARK_GREEN,
    DARK_AQUA,
    DARK_RED,
    DARK_PURPLE,
    GOLD,
    GRAY,
    DARK_GRAY,
    BLUE,
    GREEN,
    AQUA,
    RED,
    LIGHT_PURPLE,
    YELLOW,
    WHITE,
    MAGIC,
    BOLD,
    STRIKETHROUGH,
    UNDERLINE,
    ITALIC,
    RESET
    ;

    fun asBukkit() : ChatColor {
        return ChatColor.valueOf(name)
    }

    operator fun plus(other: String) : String {
        return toString()+other
    }

    override fun toString(): String {
        return name.lowercase()
    }
}